dqvm/src/utils.lisp

```diff
@@ -0,0 +1,37 @@
+;;;; src/utils.lisp
+;;;;
+;;;; Author: Juan M. Bello-Rivas
+;;;;         Robert Smith
+;;;;         Lauren Capelluto
+
+(in-package #:dqvm2)
+
+(defun error-missing-initform (symbol)
+  (error "You must specify ~a" (prin1-to-string symbol)))
+
+(defun instruction->string (instruction)
+  "Return string representation of INSTRUCTION."
+  (with-output-to-string (string)
+    (quil:print-instruction instruction string)))
```

Comment:
There is `quil::print-instruction-to-string`. Don't see why it couldn't be exported.

Reply:
use `format nil "~/quil:instruction-fmt/"`